app/src/full/java/io/homeassistant/companion/android/settings/views/SettingsWearFavoritesView.kt

```diff
@@ -0,0 +1,140 @@
+package io.homeassistant.companion.android.settings.views
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.Checkbox
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.HelpOutline
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import io.homeassistant.companion.android.R
+import io.homeassistant.companion.android.common.data.integration.Entity
+import io.homeassistant.companion.android.util.previewEntityList
+import io.homeassistant.companion.android.util.previewFavoritesList
+
+const val WEAR_DOCS_LINK = "https://companion.home-assistant.io/docs/wear-os/wear-os"
+val supportedDomains = listOf(
+    "input_boolean", "light", "switch", "script", "scene"
+)
+
+@Composable
+fun LoadWearFavoritesSettings(
+    entities: Map<String, Entity<*>>,
+    favoritesList: List<String>,
+    onEntitySelected: (Boolean, String, Activity) -> Unit,
+    isFavoriteSelected: (Int) -> Boolean,

```

Comment:
Why do we need to pass this function?  Can't we determine inline?

Reply:
I cant recall, I have switched to inline :)